### PR TITLE
Bruin CLI - Continue from last failure

### PR DESCRIPTION
**Pull request by y-bruin**

### CODE REVIEW

File: webview-ui/src/components/asset/AssetGeneral.vue [hunk 1]
```diff
@@ -512,6 +520,19 @@ function runAssetWithDownstream() {
   });
 }
 
+
+
+function runPipelineWithContinue() {
+  let payload = getCheckboxChangePayload();
+  payload = payload + " --downstream" + " --environment " + selectedEnv.value + " --continue";
```
Comment (karakanb) [inline webview-ui/src/components/asset/AssetGeneral.vue:527]: why do we need these values for `--continue`, shouldn't it be getting it from the previous state?

File: src/panels/BruinPanel.ts [hunk 2]
```diff
@@ -355,6 +355,25 @@ export class BruinPanel {
               "bruin"
             );
 
+            setTimeout(() => {
+              this._panel.webview.postMessage({
+                command: "runCompleted",
+                message: "",
+              });
+            }, 1500);
+            break;
```
Comment (karakanb) [inline src/panels/BruinPanel.ts:364]: what is this setTimeout doing here?

File: webview-ui/src/components/asset/AssetGeneral.vue [hunk 3]
```diff
@@ -176,6 +176,14 @@
                         Run the whole pipeline
                       </vscode-button>
                     </MenuItem>
+                    <MenuItem key="run-with-continue" v-slot="{ active }">
+                      <vscode-button
+                        class="block text-editor-fg border-0 rounded-sm w-full text-left text-2xs hover:bg-editor-button-hover-bg hover:text-editor-button-fg bg-editorWidget-bg"
+                        @click="runPipelineWithContinue"
+                      >
+                        Run with continue
```
Comment (karakanb) [inline webview-ui/src/components/asset/AssetGeneral.vue:184]:
```suggestion
                        Continue from last failure
```

File: src/panels/BruinPanel.ts (hunk 2 above)
Comment (DjamilaBaroudi) [reply src/panels/BruinPanel.ts:364]: @y-bruin You can remove the `runCompleted` command as it doesn't send any information to the UI. The `setTimeout` was originally added to ensure enough time for the terminal to be created and the command to run, but it's no longer necessary.

**Pull request closed without merging**